### PR TITLE
Added merge_sorted_lists func and tests

### PR DESCRIPTION
**Pull request by angela100743**

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
```
Comment (ewatki) [inline functions/merge_sorted_lists.py:2]: Nice job!

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
+
+def test_for_1_and_2():
```
Comment (ewatki) [inline tests/merge_sorted_lists_test.py:5]: All tests are passing!

**Review by ewatki — COMMENTED** [review]
Overall, good work!

File: functions/merge_sorted_lists.py (hunk 1 above)
Comment (MotifNoticer) [reply functions/merge_sorted_lists.py:2]: Good work, this is a concise, working solution!

File: tests/merge_sorted_lists_test.py (hunk 2 above)
Comment (MotifNoticer) [reply tests/merge_sorted_lists_test.py:5]: Tests are written correctly and all are passing.

**Review by MotifNoticer — APPROVED** [review]

**Pull request merged**